### DISCUSSION TITLE
Consider add mirror feature ?

### DISCUSSION THREAD

**Discussion by Alceatraz** [sdkman/sdkman-cli#1151] [Ideas]
Hello:

Such as #1059 #1029. Or large scale deployment (A lot of developer/VMs) will cause a large data fetch.

Please allow me describe this feature if it become true, Then:

- Install by mirror (Http server like Nginx) hosted script, and set mirror server address `curl -s "https://get.sdkman.io.my-mirror.local" | SDKMAN-MIRROR="get.sdkman.io.my-mirror.local" bash`
- Then use `sdk cache java xxx` at server to cache packages.
- Client `sdk i java xxx` will fetch package from mirror server.

To achive thie feature, May be need url parse function and new sub command.

So, Is it possible to add this feature?

**Comment by justqyx** [comment]
+1
